grafana-plugin/src/containers/ServiceNowConfigDrawer/ServiceNowConfigDrawer.tsx

```diff
@@ -111,24 +110,6 @@ export const ServiceNowConfigDrawer: React.FC<ServiceNowConfigurationDrawerProps
               <ServiceNowStatusSection />
             </div>
 
-            <div className={styles.border}>
-              <VerticalGroup>
-                <HorizontalGroup spacing="xs" align="center">
-                  <Text type="primary" strong>
-                    Labels Mapping
-                  </Text>
-                  <Icon name="info-circle" />
-                </HorizontalGroup>
-
-                <Text>
-                  Description for such object and{' '}
-                  <a href={'#'} target="_blank" rel="noreferrer">
-                    <Text type="link">link to documentation</Text>
-                  </a>
-                </Text>
-              </VerticalGroup>
-            </div>
-
```

Comment:
This is not a thing atm, so removing it completely (so settings are easier to navigate)